### PR TITLE
1.14/1811 refactoring signup and edit reg screens

### PR DESCRIPTION
**Pull request by lauragurney**
The voucher collectors (primary carer/secondary carer etc) works functionally now. All 3 columns on these pages should display properly now too. Middle column won't work functionally due to missing age DOB stuff. Tests for these pages will fail because I need to change them.

### CODE REVIEW

File: resources/views/store/create_registration.blade.php [hunk 1]
```diff
@@ -1,220 +1,34 @@
...
-        });
-        //remove invalid class when input is selected/tabbed to
-        $('#privacy-statement, #carer').on('click focus', function() {
-            $(this).removeClass("invalid");
-            // Remove relevent error message
-            var spanclass = $(this)[0].id + '-span';
-            $('#' + spanclass).addClass('collapsed');
-        });
-    </script>
+<script src="{{ asset('store/js/create_registration.js') }}"></script>
```
Comment (charlesstrange2) [inline resources/views/store/create_registration.blade.php:32]: look at how mix is doing the copying and mangling of javascript into `public/`

File: resources/views/store/edit_registration.blade.php [hunk 2]
```diff
@@ -4,304 +4,32 @@
...
-        });
-
-        $('#cancel').click(function (e) {
-            $('#expandable').addClass('collapsed');
-            $('#leaving').removeClass('expanded');
-            e.preventDefault();
-        });
-
-    </script>
+<script src="{{ asset('store/js/edit_registration.js') }}"></script>
```
Comment (charlesstrange2) [inline resources/views/store/edit_registration.blade.php:33]: similarly, look at how mix is mangling javascript into the public directories.

File: resources/views/store/partials/other_infoSP.blade.php [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+        </div>
+        <button class="long-button submit" type="submit" formnovalidate>Save Changes</button>
+        <a href="{{ route("store.registration.voucher-manager", ['registration' => $registration ]) }}" class="link">
+            <div class="link-button link-button-large">
+                <i class="fa fa-ticket button-icon" aria-hidden="true"></i>Go to voucher manager
+            </div>
+        </a>
+
+        @if (!isset($registration->family->leaving_on) )
+        <button class="remove long-button" type="button">Remove this family</button>
```
Comment (charlesstrange2) [inline resources/views/store/partials/other_infoSP.blade.php:48]: check for "family"

**Review by charlesstrange2 — CHANGES_REQUESTED** [review]
check for family names etc.

File: resources/views/store/partials/other_infoSP.blade.php (hunk 3 above)
Comment (lauragurney) [reply resources/views/store/partials/other_infoSP.blade.php]: Done

File: resources/views/store/create_registration.blade.php (hunk 1 above)
Comment (lauragurney) [reply resources/views/store/create_registration.blade.php:32]: As discussed, we will be leaving this well alone for now!

**Pull request merged**